### ISSUE TITLE
_sanitize_recursive problem

### ISSUE THREAD

**Issue by anteksiler** [valendesigns/option-tree#702]
I am getting cannot redeclare on the code block below:
```
function _sanitize_recursive( $values = array() ) {
$result = array();
foreach ( $values as $key => $value ) {
	if ( ! is_object( $value ) ) {
		if ( is_scalar( $value ) ) {
			$result[ $key ] = sanitize_textarea_field( $value );
		} else {
			$result[ $key ] = _sanitize_recursive( $value );
		}
	}
}
```
Couldn't figure it out why.

**Comment by victoras** [comment]
I get the same issue, when using Google Fonts!

**Comment by Ninetheme** [comment]
same +1

is there any custom fix?

**Comment by valendesigns** [comment]
This was fixed in 2.7.1, thanks for reporting the issue.
